QueueInteropTransport.php

```diff
@@ -69,7 +69,7 @@ public function receive(callable $handler): void
 
         while (!$this->shouldStop) {
             try {
-                if (null === ($message = $consumer->receive($this->options['receiveTimeout'] ?? 0))) {
+                if (null === ($message = $consumer->receive($this->options['receiveTimeout'] ?? 30))) {
```

Comment:
FYI: 30 means 30 milliseconds (not seconds).

Reply:
@makasim thanks, changed.